samples/contrib/aws-samples/ground_truth_pipeline_demo/README.md

```diff
@@ -34,25 +34,92 @@ client_ID = App client
 
 > Note : Once you start a run on the pipeline you will receive the ground_truth labeling jobs at "Labeling portal sign-in URL" link 
 
-## SageMaker permission
```

Comment:
I don't suggest you totally remove this. IRSA is only working for EKS. Not all uses are using EKS, there's tons of self managed K8s clusters. You can use IRSA as default but leave all the options there.

Reply:
Is it possible to keep `aws_secret` as an additional option?  Having the ability to run this example on non-EKS clusters would be hugely valuable.  I can help with testing it on a self-hosted cluster next week.

Reply:
I think thats what Jiaxin is suggesting as well. We should have both options available. EKS customer can get benefit using IRSA (AWS IAM role for service account) and non EKS customer can benefit using secret. 

@akartsky  Make the IRSA option as recommended and keep the `secret` option as well.

Reply:
done